README.md

```diff
@@ -73,9 +73,9 @@ import xxhash from "xxhash-wasm";
 xxhash().then(hasher => {
   const input = "The string that is being hashed";
   // 32-bit version
-  hasher.h32(input); // ee563564
+  hasher.h32ToString(input); // ee563564
   // 64-bit version
-  hasher.h64(input); // 502b0c5fc4a5704c
+  hasher.h64ToString(input); // 502b0c5fc4a5704c
```

Comment:
Put the output in the comments in quotes, e.g. `"ee563564"`, to make it clearer that it's a string.

For the version below where numbers are returned, maybe also add `(decimal representation)` or `(BigInt)` or something along these lines to clarify that those are not hex representations and hence differ from the string values above.

Also the only difference between the two code blocks was the `async`/`await`, that should probably kept in sync and the string version in a separate one, explaining the availability of the `h**ToString` methods as a convenience to get the hexadecimal representation.

Reply:
👍 Good feedback.